src/components/axes/axis.ts

```diff
@@ -30,15 +30,14 @@ module Plottable {
     private _rescaleCallback: ScaleCallback<Scale<D, number>>;
 
     /**
-     * Constructs an axis. An axis is a wrapper around a scale for rendering.
+     * Constructs an Axis.
+     * An Axis is a visual representation of a Scale.
```

Comment:
An Axis is a visual representation of a Scale through a baseline and ticks.

?